website/content/en/preview/AWS/provisioning.md

```diff
@@ -6,6 +6,36 @@ description: >
   Learn AWS-specific Karpenter provisioning settings
 ---
 
+Provisioner settings specific to Karpenter for the AWS cloud provider are described here.
+The following example shows optional and required settings for a Karpenter provisioner for AWS:
+
+
+```yaml
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:                      
+    - key: karpenter.sh/capacity-type         # optional, set to on-demand by default

```

Comment:
This isn't really cloud provider specific, we may want to leave it off entirely for this part since it's explained in the provisioner API docs: https://karpenter.sh/v0.10.0/provisioner/#:~:text=%2D%20key%3A%20%22karpenter.sh/capacity%2Dtype%22%20%23%20If%20not%20included%2C%20the%20webhook%20for%20the%20AWS%20cloud%20provider%20will%20default%20to%20on%2Ddemand